imls-frontend/src/store/store.js

```diff
@@ -1,16 +1,15 @@
 import { reactive, computed, readonly } from "vue";
 
-// todo: update when the backend has a real host
-const BACKEND_BASEURL = `${window.location.protocol}//${window.location.hostname}:3000`;
+const BACKEND_BASEURL = import.meta.env.VITE_BACKEND_BASEURL;
```

Comment:
A note to reviewers -- Vite, our build process, requires that all ENV vars that are eligible to be passed into the app are prefixed with "VITE_", so that we don't leak secrets, etc. So if you ever want to add more env variables for use in the front end, make sure they start with that, or you won't see them!